### PR TITLE
I literally recommended the first things that entered my head.

### LINKED ISSUE #8: joaltaylor stuff to do in Sydney, Australia
If you like cats.

- [ ] Places to eat
- [ ] Stuff to do

### PR DESCRIPTION
**Pull request by joaltaylor**
This closes #8 

Added a file that suggests that you spend an hour petting kitties. It's good for your blood pressure.

### CODE REVIEW

File: joaltaylor-sydney.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Sydney, Australia
+
+## Fud
+
+Eat lunch at The Rocks.
+
+## Enjoy
+
+Catmosphere is a cat cafe where you can have a coffee and pet kitties for an hour.
```
Comment (rbmwork) [inline joaltaylor-sydney.md]: @jgancisra  You serious?

Comment (rbmwork) [inline joaltaylor-sydney.md]: @joaltaylor  You serious, an hour with cats?

Comment (rbmwork) [inline joaltaylor-sydney.md]: @joaltaylor  You serious, an hour with cats?

File: joaltaylor-sydney.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Sydney, Australia
+
+## Fud
+
+Eat lunch at The Rocks.
```
Comment (rbmwork) [inline joaltaylor-sydney.md]: Great idea

File: joaltaylor-sydney.md (hunk 1 above)
Comment (joaltaylor) [reply joaltaylor-sydney.md]: I may be biased.

Comment (rbmwork) [reply joaltaylor-sydney.md]: OK. We'll let it pass.

File: joaltaylor-sydney.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+## Enjoy
+
+Catmosphere is a cat cafe where you can have a coffee and pet kitties for an hour.
+Dog people should go to Catmosphere and spend an hour learning to love kitties.
```
Comment (rbmwork) [inline joaltaylor-sydney.md:12]: I'll have to man up.

**Review by rbmwork — CHANGES_REQUESTED** [review]

**Review by rbmwork — APPROVED** [review]

**Pull request merged**